setup.cfg

```diff
@@ -26,26 +26,23 @@ description = Software development kit for open source D-Wave tools
 
 [options]
 install_requires =
-    dimod==0.12.15
-    dwave-cloud-client==0.12.0
+    dimod==0.12.16
+    dwave-cloud-client==0.12.1
+    dwave-gate==0.3.2
     dwave-greedy==0.3.0
     dwave-hybrid==0.6.11
     dwave-inspector==0.5.0.post0
     dwave-neal==0.6.0
     dwave-networkx==0.8.15
-    dwave-optimization==0.1.0
-    dwave-preprocessing==0.6.5
-    dwave-samplers==1.2.0
+    dwave-optimization==0.2.0
+    dwave-preprocessing==0.6.6
+    dwave-samplers==1.3.0
     dwave-system==1.25.0
```

Comment:
We could also release new dwave-system, but since there're only minor test and docs improvements, we can skip it this time as well.